### PR TITLE
Latest changes for cdc stressor: enable batching

### PR DESCRIPTION
**Pull request by aleksbykov**
cdc stressor tool was updated with new command parameter and flow of execution.
Added appropriate changes to sct

Cdc stressor now support batching: Read changes from several streams in a batch.
Added collecting log file of cdc stressor tool

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by bentsi** [comment]
please resolve conflicts

**Comment by aleksbykov** [comment]
@bentsi Done

File: test-cases/performance/perf-regression-throughput-cdc-mixed-poll-batching.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+store_perf_results: true
+append_scylla_args: '--blocked-reactor-notify-ms 50'
+
+
+append_scylla_yaml: |
+  experimental_features:
+    - cdc
+
+send_email: true
+email_recipients: ['alex.bykov@scylladb.com']
```
Comment (bentsi) [inline test-cases/performance/perf-regression-throughput-cdc-mixed-poll-batching.yaml:30]: change to qa

Comment (aleksbykov) [reply test-cases/performance/perf-regression-throughput-cdc-mixed-poll-batching.yaml]: fixed

File: test-cases/performance/perf-regression-throughput-cdc-mixed-poll-batching.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+test_duration: 500
```
Comment (bentsi) [inline test-cases/performance/perf-regression-throughput-cdc-mixed-poll-batching.yaml:1]: can be a bit less, like 150, right?

File: jenkins-pipelines/perf-regression-cdc-mixed-throughput-60min.jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+perfRegressionParallelPipeline(
+    params: params,
+
+    backend: "aws",
+    aws_region: "us-east-1",
+    test_name: "performance_regression_cdc_test.PerformanceRegressionCDCTest",
+    test_config: "test-cases/performance/perf-regression-throughput-cdc-mixed-poll-batching.yaml",
+    sub_tests: ["test_mixed_throughput"],
+
+    timeout: [time: 550, unit: "MINUTES"]
```
Comment (bentsi) [inline jenkins-pipelines/perf-regression-cdc-mixed-throughput-60min.jenkinsfile:15]: 60 min != 550

**Pull request merged**